### PR TITLE
exchange:ftx: add optimized access for submit order

### PR DESCRIPTION
**Pull request by austin362667**
#998

### CODE REVIEW

File: pkg/exchange/ftx/ftxapi/client.go [hunk 1]
```diff
@@ -66,6 +67,28 @@ func NewClient() *RestClient {
...
+			Timeout: defaultHTTPTimeout,
+		},
+	}
+
+	/*
+		client.AccountService = &AccountService{client: client}
+		client.MarketDataService = &MarketDataService{client: client}
+		client.TradeService = &TradeService{client: client}
+		client.BulletService = &BulletService{client: client}
+	*/
```
Comment (c9s) [inline pkg/exchange/ftx/ftxapi/client.go:88]: remove unused code thanks

File: pkg/exchange/ftx/ftxapi/client.go [hunk 2]
```diff
@@ -22,6 +22,7 @@ import (
 
 const defaultHTTPTimeout = time.Second * 15
 const RestBaseURL = "https://ftx.com/api"
+const OptimizedRestBaseURL = "https://api.ftx.com"
```
Comment (c9s) [inline pkg/exchange/ftx/ftxapi/client.go:25]: Is this for everyone?

**Review by c9s — CHANGES_REQUESTED** [review]
need more info

**Comment by austin362667** [comment]
can simply override env var with the new optimized access HTTP  endpoint

**Pull request closed without merging**